### PR TITLE
Spanish translation of ch06-00-enums.md and  ch06-01-defining-an-enum.md (Partial)

### PR DESCRIPTION
**Pull request by dbucci24**
This is my first contribution to the project, I hope you like it to continue collaborating! :)

### CODE REVIEW

**Comment by thecodix** [comment]
Hey @dbucci24, welcome to this project, may I ask (like I did for another contributor) you to try and follow the format of the english version on all paragraphs and cut each line around the same width? (cant recall if its at 120 characters)

That way it will ease review/edition tasks no matter the screen used

Thanks A LOT for contributing!

**Comment by dbucci24** [comment]
@thecodix ok, I've fixed it, should I upload another pull request?

**Comment by thecodix** [comment]
Hey, you can fit all in this same PR, please check again the latest changes, since (after you merged from master) there is now same text in both spanish and english in some parts

**Comment by thecodix** [comment]
Sorry for the mess, I should have done this merge some time ago :(

**Comment by dbucci24** [comment]
wow, everything broke ...

**Comment by thecodix** [comment]
Please let me know if you find issues merging, ideally you can override all inside your folders with 'accept theirs' and take from github (or from a stash) your changes

**Comment by dbucci24** [comment]
@thecodix  
Can I delete this pull request and open another with the things fixed?

**Comment by dbucci24** [comment]
@thecodix  can you check it from the first files (with the line width was uncorrected) ?, ignore the changes I made, that broke ... for the next translations I will do it with the correct line width

File: second-edition/src/ch06-01-defining-an-enum.md [hunk 1]
```diff
@@ -97,23 +102,31 @@ let loopback = IpAddr {
...
+`address`, por lo que la variante ahora está asociada con el valor.
+
+
+Podemos representar el mismo concepto de una forma más concisa
+usando solo una enumeración en vez de una enumeración como parte
+de una estructura al poner datos directamente en cada variante de
+enumeración. Esta nueva definición de enumeración `IpAddr` dice
+que tanto las variantes `V4` y `V6` tendrán asociados valores `String`:
+=======
 We can represent the same concept in a more concise way using just an enum,
```
Comment (thecodix) [inline second-edition/src/ch06-01-defining-an-enum.md]: Please look at this duplicated code after the merge, this is translated just above

File: second-edition/src/ch06-01-defining-an-enum.md [hunk 2]
```diff
@@ -194,21 +214,28 @@ enum Message {
...
-This enum has four variants with different types:
+Esta enumeración tiene cuatro variantes con diferentes tipos:
 
+
+* `Quit` no tiene datos asociados con el del todo.
+* `Move` incluye una estructura anónima dentro de ella.
+* `Write` incluye un solo `String`.
+* `ChangeColor` incluye tres `i32`.
+=======
 * `Quit` has no data associated with it at all.
```
Comment (thecodix) [inline second-edition/src/ch06-01-defining-an-enum.md]: This part seems duplicated as well

Comment (dbucci24) [reply second-edition/src/ch06-01-defining-an-enum.md]: @thecodix 
How can I solve that? I can not delete the text in English .. please help me..

**Comment by thecodix** [comment]
I'll merge the request, you can review if there's missing parts in english later, after doing a pull :)

**Comment by dbucci24** [comment]
@thecodix thanks, sorry for the problems, when I tried to modify the width of the lines everything was disorganized, but the next pull request will have no problems :)

**Pull request merged**